apps/remix-ide/src/blockchain/providers/injected.ts

```diff
@@ -29,16 +33,16 @@ export class InjectedProvider {
   }
 
   getGasPrice (cb) {
-    this.executionContext.web3().eth.getGasPrice(cb)
+    this.executionContext.web3().eth.getGasPrice().then((result => cb(null, result)))
   }
 
   signMessage (message, account, _passphrase, cb) {
     const messageHash = hashPersonalMessage(Buffer.from(message))
     try {
       message = isHexString(message) ? message : Web3.utils.utf8ToHex(message)
-      this.executionContext.web3().eth.personal.sign(message, account, (error, signedData) => {
+      this.executionContext.web3().eth.personal.sign(message, account).then((error, signedData) => {
```

Comment:
This change can be an async await call, since it is in a try catch statement. That way we eliminate duplicate catch functions. Just a suggestion, it doesn't need to be changed.